woocommerce/woocommerce-gateway-stripe
# feat: payment request page data persist

# Changes proposed in this Pull Request:
Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1813

Allows persisting the data on the "Express checkouts" page.

I moved the files from `client/settings/payment-requests/*` to `client/entrypoints/payment-request-settings/*`, and ensured that the entrypoint `payment_requests_customizer` matches the file name more closely (so that it's easier to find).

**PLEASE NOTE:** the button preview is still not working, because the account keys are missing - separate ticket for it: https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1846


# Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&area=payment_requests
- Change some of the values
- Click the "Save changes" button
- Data is persisted
- Refresh the page
- The same values you selected are now displayed
